public/js/components/tests/utils.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+const React = require("react");
+const { createElement, createFactory } = React;
+const ReactDOM = require("react-dom");
+
+const renderComponentFromFixture =
+      require("../../test/utils/renderComponentFromFixture");
+
+function getSandbox() {
+  let $el = document.querySelector("#sandbox");
+  if (!$el) {
+    const attribute = document.createAttribute("id");
+    attribute.nodeValue = "sandbox";
+    $el = document.createElement("div");
+    $el.setAttributeNode(attribute);;
+    document.body.appendChild($el);
+  }
+
+  $el.innerHTML = "";
+
+  return $el;
+}
```

Comment:
we need to create the sandbox element the first time because we don't have control over karma's runner `html` file